pkg/model/const.go

```diff
@@ -203,8 +203,8 @@ const (
 	// OktetoSkipCleanupEnvVar defines the okteto binary that should be used
 	OktetoSkipCleanupEnvVar = "OKTETO_SKIP_CLEANUP"
 
-	// OktetoUserEnvVar defines the user is using okteto
-	OktetoUserEnvVar = "OKTETO_USER"
+	// OktetoUserEnvVar defines the user using okteto
+	OktetoUserEnvVar = "OKTETO_USER_ID"
```

Comment:
why are we changing this env var? `OKTETO_USER` is being used in different integration tests and defined at docs for the tests. Also is being added as env to the circleci jobs.

Reply:
@teresaromero great catch! I don't know why my vscode didn't show me the references.

This comment also helped me see that there's a better envvar for this, `OKTETO_USERNAME`. I'll update the code and ping you again